app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -203,5 +204,20 @@ def search_result_document(search_params)
         raise CanCan::AccessDenied.new(nil, :show) if document_list.empty?
         document_list.first
       end
+
+      def authorize_workflow_role!
```

Comment:
It seems to me that we should have `user_cannot_access_due_to_workflow?` in Ability (or at least not the controller).  Since CanCan is doing authorization with exceptions, should we be following that pattern with `authorize_workflow_role!` (and do `rescue_from WorkflowException, with: :my_exception handler?`